### DISCUSSION TITLE
2009/09/15/%E5%8E%9F%E7%A8%BF%E4%B8%AD%E7%9A%84%E5%89%8D%E8%A8%80

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#279] [Announcements]
# 2009/09/15/%E5%8E%9F%E7%A8%BF%E4%B8%AD%E7%9A%84%E5%89%8D%E8%A8%80

交稿之后，出版社的编辑又帮我检查修改了数遍，改正了不少原稿中的错误，语言风格也比原来严谨些。比如前言部分，原稿中我是使用第一人称来记录自己的一些经历和感想的，在正式出版的书中则改为了第三人称。我觉得这也不是很重要的部分，不过前几天有人问起，所以我就把我的原稿也贴上来吧：

https://blog.qizhen.xyz/2009/09/15/%E5%8E%9F%E7%A8%BF%E4%B8%AD%E7%9A%84%E5%89%8D%E8%A8%80

**Comment by ruanqizhen** [comment]
之前博客下的留言：

冬艳
2009年09月15日 23:50 编辑
哈哈,沙发~第一次读到这么详细真诚的前言~

回复
